deep-learning/_posts/2019-01-22-FCN_MultiStepRL_ImageProcessing.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: review
+title: "Fully Convolutional Network with Multi-Step Reinforcement Learning for Image Processing"
+tags: deep-learning reinforcement-learning CNN denoising
+author: "Antoine Théberge"
+cite:
+    authors: "Ryosuke Furuta, Naoto Inoue, Toshihiko Yamasaki"
+    title:   "Fully Convolutional Network with Multi-Step Reinforcement Learning for Image Processing"
+    venue:   "Arxiv"
+pdf: "https://arxiv.org/abs/1811.04323"
+---
+
+## Introduction
+
+Reinforcement learning methods are typically used to control robots and have only recently been used for image processing purposes. The authors of the paper do the latter by assigning an agent per pixel, where the agent changes the pixel value by taking an action. They apply the method to three image processing tasks: Image denoising, image restoration and local color enhancement. They claim to achieve comparable or better performances than state-of-the-art methods.
+
+## Formally
```

Comment:
May be `Method` is better as a section heading.

Reply:
Indeed